metricbeat/module/php_fpm/_meta/fields.yml

```diff
@@ -0,0 +1,11 @@
+- key: php_fpm
+  title: "php_fpm"
+  description: >
+    PHP-FPM server status metrics collected from PHP-FPM.
+  fields:
```

Comment:
Can you add here the `experimental[]` flag? We do that for all new modules so we can still break compatibility after the first release. This might be a beta flag soonish.

Also please add `short_config: false` so it does not show up in the short config. See https://github.com/elastic/beats/blob/master/metricbeat/module/prometheus/_meta/fields.yml#L7 as example.